pkg/controller/common/util.go

```diff
@@ -329,6 +329,9 @@ const (
 
 	// AnnCdiCustomizeComponentHash annotation is a hash of all customizations that live under spec.CustomizeComponents
 	AnnCdiCustomizeComponentHash = AnnAPIGroup + "/customizer-identifier"
+
+	// AnnCreatedForDataVolume stores the UID of the datavolume that the PVC was created for
+	AnnCreatedForDataVolume = AnnAPIGroup + "/createdForDataVolume"
```

Comment:
Would this work on DVs from previous versions? I recall this argument from https://github.com/kubevirt/containerized-data-importer/pull/3105

Reply:
All pvcs without this annotation (created prev version) are adoptable